src/controllers/user-controller.js

```diff
@@ -0,0 +1,19 @@
+const { mapQueryToUserRequest } = require('../utils');
+const { userService, roverPhotoService } = require('../services');
+
+const postUser = async (req, res, next) => {
+    const request = mapQueryToUserRequest(req.query);
+
+    try {
+        // eslint-disable-next-line no-unused-vars
+        const userDataProcessed = await userService.processUserData(request);
```

Comment:
if you don't need the result of the promise, you don't have to create the variable
```js
try {
  await userService.processUserData(request);
  ...
```

Reply:
Thank you. This makes sense. Updated.